### PR TITLE
yaroslav.hayek/sorting

### PR DESCRIPTION
**Pull request by horianyn**
I`ve changed a little our home work exercise by adding several new features:

I`ve added sorting using Array.Sort
I`ve compare different types of sorting algorithms by time they spend to finish sorting

### CODE REVIEW

File: Sorting/Sorting/SortingMethods.cs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                    break;
+                default:
+                    break;
+            }
+            return array;
+        }
+
+        public static double[] BubleSort(double[] array, Direction direction)
+        {
+            for (int i = 0; i < array.Length - 1; i++)
```
Comment (kievzenit) [inline Sorting/Sorting/SortingMethods.cs]: Lines 146-157 are useless

**Review by kievzenit — CHANGES_REQUESTED** [review]

File: Sorting/Sorting/SortingMethods.cs (hunk 1 above)
Comment (horianyn) [reply Sorting/Sorting/SortingMethods.cs]: you right mate, thank you

Comment (kievzenit) [reply Sorting/Sorting/SortingMethods.cs]: You're welcome!

**Review by kievzenit — APPROVED** [review]

**Review by SergiySimchuk — APPROVED** [review]
теж хотів дерево заімплементувати, але  часу не знайшлося. А так білднув твоє рішення і подивився) Дякую)

File: Sorting/Sorting/SortingMethods.cs [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    }
+                    else
+                    {
+                        this.Right.Add(value);
+                    }
+                }
+            }
+        }
+
+        private static void ascendingTreeSorting(Node node, ref List<double> list)
```
Comment (Artya) [inline Sorting/Sorting/SortingMethods.cs:44]: code style problem in the name

File: Sorting/Sorting/SortingMethods.cs [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+            list.Add(node.Value);
+
+            if (node.Right != null)
+            {
+                ascendingTreeSorting(node.Right, ref list);
+            }
+        }
+
+        private static void descendingTreeSorting(Node node, ref List<double> list)
```
Comment (Artya) [inline Sorting/Sorting/SortingMethods.cs:59]: code style problem in the name

**Review by Artya — APPROVED** [review]

File: Sorting/Sorting/SortingMethods.cs (hunk 3 above)
Comment (horianyn) [reply Sorting/Sorting/SortingMethods.cs:59]: Understand. Thank you. My fault. It is metyhod!

**Pull request merged**